lib/readline.js

```diff
@@ -120,8 +119,8 @@ function Interface(input, output, completer, terminal) {
   this.input = input;
   this.historySize = historySize;
   this.removeHistoryDuplicates = !!removeHistoryDuplicates;
-  this.crlfDelay = Math.max(kMincrlfDelay,
-                            Math.min(kMaxcrlfDelay, crlfDelay >>> 0));
+  this.crlfDelay = crlfDelay === Infinity ?
```

Comment:
@refack  This is because `crlfDelay` is number type (not integer), `crlfDelay  >>> 0` convert float to integer, but `Infinity >>> 0` is 0.